### DISCUSSION TITLE
Upcoming priority tickets

### DISCUSSION THREAD

**Discussion by ryanluker** [ryanluker/vscode-coverage-gutters#461] [General]
Just to get a bit of perspective on the next milestone priorities for [Milestone 2.13.0](https://github.com/ryanluker/vscode-coverage-gutters/milestone/47) we will try to focus on the top 2-3 most requested issues (by thumbs up / general usefulness)!

Open to other items also being worked on and or prioritized too.

https://github.com/ryanluker/vscode-coverage-gutters/issues/236 Starting the watch on open, should be fairly quick and doable.
https://github.com/ryanluker/vscode-coverage-gutters/issues/151 File tree coverage, might be a bit of work, thus the current `3.0`.
https://github.com/ryanluker/vscode-coverage-gutters/issues/392 Optional live preview dependency, nice bug to cleanup.

**Comment by ryanluker** [comment]
@mattseddon @Gerrit0 Many thanks for helping with the priority tickets!
I am going to close this discussion as the file tree coverage is meant for `3.0` which is a bit further out.

**Comment by mattseddon** [comment]
LOL, I didn't even know this discussion was a thing. I hit #236 completely by luck.

**Reply by ryanluker** [reply]
Excellent! My triage must have hit on the best outstanding tickets 😂.

**Reply by Gerrit0** [reply]
Same here! I never use the preview functionality so every once in a while I'd look at extensions and wonder why that live server extension was there!
